dpgen/generator/run.py

```diff
@@ -730,7 +748,13 @@ def run_train(iter_index, jdata, mdata):
             "training_init_model, training_init_frozen_model, and training_finetune_model are mutually exclusive."
         )
 
-    train_command = mdata.get("train_command", "dp")
+    train_command = mdata.get("train_command", "dp").strip()
+    # assert train_command == "dp", "The 'train_command' should be 'dp'"     # the tests should be updated to run this command
+    if suffix == ".pb":
+        train_command += " --tf"
```

Comment:
This breaks support for old versions. This line can be removed, as `--tf` is the default behavior.